src/main/java/duke/storage/Storage.java

```diff
@@ -0,0 +1,135 @@
+package duke.storage;
+
+import duke.tasks.*;
+import duke.taskList.TaskList;
+
+import java.io.File;
+import java.io.IOException;
+import java.io.FileWriter;
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+
+/**
+ * A Storage class that creates a storage object which
+ * Saves the Arraylist of Tasks into a .txt file.
+ * Loads the Arraylist of Tasks from a .txt file into a ArrayList Object
+ */
+
+public class Storage {
+    public static String DIRECTORY;
+
+    /**
+     * Creates a storage object
+     *
+     * @param directory, the path of storage file
+     */
+
+    public Storage(String directory) {
+        this.DIRECTORY = directory;
+    }
+
+
+    /** Saves the list of Tasks in the ArrayList object
```

Comment:
Perhaps the opening /** can occupy a whole line on its own and description can be one line down.